components/gitpod-protocol/go/reconnecting-ws.go

```diff
@@ -218,7 +218,7 @@ func (rc *ReconnectingWebsocket) connect(ctx context.Context) *WebsocketConnecti
 		if resp != nil {
 			statusCode = resp.StatusCode
 		}
-		rc.log.WithField("url", rc.url).WithField("headers", rc.reqHeader).Info("websocket handshake")
+		rc.log.WithField("url", rc.url).Info("websocket handshake")
```

Comment:
I think this completely, since it does not add new info compare to next error message. It is also misleading, websocket handshake is only one part of establishing connection. I am not sure what `websocket handshake` supposed to mean.